### PR TITLE
GCP project switcher plugin

### PR DESCRIPTION
**Pull request by danthelion**
Plugin so show current active GCP project and a list to active others.

### CODE REVIEW

File: GCP/gcp-project-switcher.1d.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# Metadata allows your plugin to show up in the app, and website.
+#
+#  <xbar.title>GCP Project Switcher</xbar.title>
+#  <xbar.version>v1.0</xbar.version>
+#  <xbar.author>Daniel Palma</xbar.author>
+#  <xbar.author.github>danthelion</xbar.author.github>
+#  <xbar.desc>See current active GCP project and easily switch between others.</xbar.desc>
```
Comment (matryer) [inline GCP/gcp-project-switcher.1d.sh:9]: Thanks for the plugin - this is one I'll use. Any chance you could add an image?

**Comment by matryer** [comment]
Hey @danthelion - could you put this into a folder called Cloud?

**Comment by danthelion** [comment]
Hey @matryer, moved the plugin to a new folder and here's a screenshot:
Screenshot 2021-05-27 at 7 56 17

**Comment by matryer** [comment]
@danthelion perfect, can you add that image URL to the `<xbar.image>` metadata please? And then we'll be good to go.

**Comment by danthelion** [comment]
@matryer Sure, done!

**Comment by matryer** [comment]
@danthelion great but sorry, one more quick thing, can you make it `https://`?

**Comment by danthelion** [comment]
@matryer yeah sure, also accidentally pasted a wrong image, so fixed that as well

**Review by sprak3000 — APPROVED** [review]

**Pull request merged**